### PR TITLE
feat: proper recomb accessory count

### LINKED ISSUE #1637: Pandora's Box shouldn't count towards recombobulated accessories
Since it can't be recombobulated but gets its rarity from the challenges completed by the player

### PR DESCRIPTION
**Pull request by metalcupcake5**
closes #1637 
image
on indigo_polecat/Pineapple

### CODE REVIEW

File: src/constants/accessories.js [hunk 1]
```diff
@@ -775,3 +776,13 @@ export const MAGICAL_POWER = {
   special: 3,
   very_special: 5,
 };
+
+const recombableAccessories = uniqueAccessories;
+
+for (const accessory of Object.keys(recombableAccessories)) {
+  if (ACCESSORIES[accessory]?.allowRecomb == false) {
```
Comment (dukio) [inline src/constants/accessories.js]: I would make this a `===`

**Review by dukio — CHANGES_REQUESTED** [review]
Only a small thing then it's good to go for me (didn't test)

**Comment by metalcupcake5** [comment]
image
dont merge until i do this

**Comment by dukio** [comment]
Also would it be more grammatically correct to use "allowsRecomb" (with s?)

**Review by dukio — APPROVED** [review]
I'll approve this, but the accessories stats need to be revisited in the future...

image

Unique accessories is the number of unique accessories owned out of all the accessories that exist, correct.

Completion should be equivalent to "this person has this progress towards having every accessory upgraded to the max (recomb, enrichment, other stuff like max gemstones on artifact of power or pandora box challenges)"

Recombobulated should be, as it was before, "this person has recombobulated X accessories out of his total accessories owned that can be recombobulated", this PR changes this to "this person has X recombobulated accessoires out of all the accessories in existance that can be recombobulated" (so not just the owned ones)

**Review by MartinNemi03 — APPROVED** [review]
lgtm, didn't test

**Pull request merged**